source/elements/oneMKL/source/domains/dft/config_params/data_layouts.rst

```diff
@@ -109,20 +113,35 @@ as expressed in :eq:`eq_idx_data_layout` corresponding to two different
 :math:`(d+1)`-tuples :math:`(m, k_{1}, k_{2}, \dots, k_d)` within valid range,
 *i.e.*, such that
 
+.. _onemkl_dft_data_layout_nonredundant_valid_range:
+
 - :math:`0 \leq m < M` wherein :math:`M` is the :ref:`number of transforms<onemkl_dft_config_number_of_transforms>`;
 
-- :math:`0 \leq k_j < n_j`, :math:`1 \leq j < d`;
+- :math:`0 \leq k_j < n_j`, :math:`1 \leq j < d` if :math:`d > 1` ;
 
 - :math:`0 \leq k_d < n_d`, for descriptors of ``COMPLEX`` template :ref:`onemkl_dft_enum_domain` or for forward data of descriptors of ``REAL`` template :ref:`onemkl_dft_enum_domain`;
 
 - :math:`0 \leq k_d \leq \lfloor\frac{n_d}{2}\rfloor`, for backward data of descriptors of ``REAL`` template :ref:`onemkl_dft_enum_domain`.
 
-Additionally, for in-place transforms (``PLACEMENT=INPLACE``), the forward and backward data
-layouts must be configured *consistently* with one another, that is,
-
-- for descriptors of ``COMPLEX`` template :ref:`onemkl_dft_enum_domain`, strides and distances must be equal in forward and backward domain;
-
-- for descriptors of ``REAL`` template :ref:`onemkl_dft_enum_domain` with :ref:`onemkl_dft_real_storage_real_real` and :ref:`onemkl_dft_conjugate_even_storage_complex_complex`, considering any :math:`(d+1)`-tuple :math:`(m, k_{1}, k_{2}, \dots, k_{d-1}, 0)` within valid range (as explicited above), the index value :eq:`eq_idx_data_layout` in forward domain must be twice the corresponding index value for the same tuple in backward domain.
+Additionally, for in-place transforms (``PLACEMENT=INPLACE``), the forward and
+backward data layouts must be configured *consistently* with one another: the
+memory address(es) of leading element(s) along the last dimension must be
+identical in forward and backward domain. Specifically, considering any
+:math:`(d+1)`-tuple :math:`(m, k_{1}, k_{2}, \dots, k_{d-1}, 0)` within valid
+range (as developed above), the memory address of the element of corresponding
+index value :eq:`eq_idx_data_layout` in forward domain (considering the
+:ref:`implicitly assumed type<onemkl_dft_config_data_element_types>` in forward
+domain) must be identical to the memory address of the element of corresponding
```

Comment:
```suggestion
domain) must point to the same memory address as
```

Reply:
I believe a memory address is compared to another memory address here (not a pointer to the memory address it is supposed to point to). Could you clarify?

Reply:
You are right. Feel free to resolve.